### PR TITLE
Update CSC programs information

### PR DESCRIPTION
**Pull request by courtney4150**
Added some updated info about CSC programs. Mostly just learning how to contribute to the wiki and using this as a starting place.

### CODE REVIEW

File: src/guides/skills/clubs.md [hunk 1]
```diff
@@ -21,23 +21,23 @@ CSC's programs include:
...
-*   Study Sessions 
-    *   Study Sessions are bring-your-own-work events where members can come to chill with other students while studying for a test or working on a project in class.
-    *   TAs for various CS classes will also be present to help assist students as well.
-*   Corporate Networking Events
-    *   Network with tech giants through CSC networking meet and greets, corporate tech talks, and company site visits!
-    *   Companies CSC has collaborated with includes:
-        *   Uber
+*   Branches
+    * Connect with a community of students interested in similar areas of technology to share resources and grow your skills! Some current branches inclube web development, iOS/Swift development, game development, and financial technology. 
+    * Not finding a branch you're looking for? Start your own!
```
Comment (EParmar18) [inline src/guides/skills/clubs.md:34]: One thing you could add here is the link to the branch creation

Comment (EParmar18) [reply src/guides/skills/clubs.md:34]: Actually looking at it how do people start their own branches lol

**Comment by EParmar18** [comment]
looks good!

File: src/guides/skills/clubs.md (hunk 1 above)
Comment (Richie78321) [reply src/guides/skills/clubs.md:34]: @EParmar18 We should make that a guide

**Review by Richie78321 — APPROVED** [review]
LGTM

File: src/guides/skills/clubs.md (hunk 1 above)
Comment (EParmar18) [reply src/guides/skills/clubs.md:34]: @Richie78321 I think we need to make concrete method for creating the branch cuz rn i think its just emailing an officer

Comment (EParmar18) [reply src/guides/skills/clubs.md:34]: then create the guide lol

**Pull request merged**